jboss-eap-config-openshift/16.0-openjdk11/module.yaml

```diff
@@ -1,6 +1,6 @@
 schema_version: 1
 name: jboss.eap.config.openshift
-version: '16.0-openjdk11'
+version: '16.0-openjdk11-kie'
```

Comment:
Why this version is changed?

Reply:
as the existing images still using the legacy branch, it is to avoid modules conflicts until we migrate them.